### PR TITLE
Added Message Objects 

### PR DESCRIPTION
**Pull request by nick2170**
as per UML: http://creately.com/diagram/is4akzct1/BuzzerWebApp
except: HostSessionCreate, ObserverUpdateMessage due to the dependencies on Session and SessionSettings.

### CODE REVIEW

File: lib/message/BuzzerStateMessage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+var AbstractMessage = require('../AbstractMessage');
+
+/**
+ * This message is from server to client and provides the buzzer state message
+ * or if not set, can be set using the setEnabled method
```
Comment (pleb) [inline lib/message/BuzzerStateMessage.js:5]: Comments should end with a full stop `.`

File: lib/message/BuzzerStateMessage.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ * BuzzerStateMessage is a subclass of AbstractMessage
+ */
+BuzzerStateMessage.prototype = Object.create(AbstractMessage.prototype);
+
+/**
+ * Set the buzzer enabled state for this message object
+ *
+ * @param {Boolean} enabled The state of the buzzer boolean
```
Comment (pleb) [inline lib/message/BuzzerStateMessage.js:26]: Extra `boolean` mentioned - not needed

Generally, I don't start a comment like with a capital, as it's usually a continuation of the sentence. For instance, 'Boolean enabled is the state of the buzzer.' or 'Boolean enabled the buzzer state.' etc.

File: lib/message/BuzzerStateMessage.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * 
+ * @public
+ */
+BuzzerStateMessage.prototype.setEnabled = function(enabled) {
+    if (enabled)
```
Comment (pleb) [inline lib/message/BuzzerStateMessage.js]: Coding standards require `{}`

File: lib/message/BuzzerStateMessage.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+ */
+InquireTeamLeaderPositionMessage.prototype.getEnabled = function() {
+    return this.data.enabled;
+};
+
+/**
+ * The type of this class
+ * @public 
+ */
+BuzzerStateMessage.prototype.type = 'BuzzerStateMessage';
```
Comment (pleb) [inline lib/message/BuzzerStateMessage.js:46]: Use @inheritdoc or don't js doc

File: .jshintrc [hunk 5]
```diff
@@ -2,14 +2,14 @@
   "esversion": 6,
   "camelcase": true,
   "curly": true,
-  "eqeqeq": true,
+  "eqeqeq": false,
```
Comment (pleb) [inline .jshintrc]: We should probably leave this to true

Comment (nick2170) [reply .jshintrc]: I agree, i took it out on mine for the moment. didn't mean to commit that!

**Pull request merged**